.github/workflows/git-town.yml

```diff
@@ -0,0 +1,24 @@
+name: Git Town
```

Comment:
Not sure `git-town.yml` is the most appropriate name for this file. How do you feel naming this file `.github/workflows/branch-stack.yml`?

Reply:
I had several motivations for the naming:

- Calling it Git Town would increase exposure, and hopefully by extension adoption. It's similar to how other tools in the ecosystem name their config files by the tool name (e.g. ESLint, Prettier, Vite, etc).
- We could potentially have more functionality we'd like to support using GitHub Actions in the future. If that was the case, adoption could be as simple as adding another job to the workflow file, since we can use descriptive names for jobs within the workflow.

IMO that outweighs the (understandably!) ambiguous name of the workflow file. What are your thoughts?

Reply:
Makes sense to me, thanks for the background!